engine/components-api/src/main/java/com/cloud/ha/HighAvailabilityManager.java

```diff
@@ -16,18 +16,36 @@
 // under the License.
 package com.cloud.ha;
 
-import java.util.List;
+import static org.apache.cloudstack.framework.config.ConfigKey.Scope.Cluster;
 
 import com.cloud.deploy.DeploymentPlanner;
 import com.cloud.host.HostVO;
 import com.cloud.host.Status;
 import com.cloud.utils.component.Manager;
 import com.cloud.vm.VMInstanceVO;
+import org.apache.cloudstack.framework.config.ConfigKey;
+
+import java.util.List;
 
 /**
  * HighAvailabilityManager checks to make sure the VMs are running fine.
  */
 public interface HighAvailabilityManager extends Manager {
+
+    ConfigKey<Long> TimeBetweenCleanup = new ConfigKey<>("Advanced", Long.class,
+        "time.between.cleanup", "86400", "Time in seconds to wait before the cleanup thread runs.",
```

Comment:
Please make the description a little more elaborate - such as cleanup of what. This is going to be used by Admins and not Devs so proper description would help.

Reply:
Also minor suggestion - mitght as well keep this in minutes? Doesn't make much use to setup in seconds.

Reply:
@anuragaw i would suggest to leave it like @chiradeep implemented it. for an infrastructure guy i think first i need to calculate thats true but i can adjust it granular.